### ISSUE TITLE
The code

### ISSUE THREAD

**Issue by floflo2607** [manami-project/anime-offline-database#22]
I just want to have the script code to help manami with update each day i think.

**Comment by manami-project** [comment]
Hi @floflo2607,
it's nice that you want to help updating the data on daily basis. I thought about that a while ago too and I came to the conclusion that this doesn't necessarily make sense from my perspective. Or rather the effort you put in doesn't pay up. Most of the time there are not many updates except for very few occasions when the seasons are about to change. But even on these very few occasions I would say that updating on a weekly base is sufficient enough. Escpecially if you look at the data this database contains. Those are data that rarely change. Sometimes the type changes from _OVA_ to _Special_ or the main title is altered. Completely new anime that haven't been aired won't be aired until the next update as well. Normally new anime are added long time before they air. So you don't really lose anything within this week.

The other thing is that I don't plan to OpenSource the code at the moment. It's not in a state that I feel comfortable with going open source. I also don't think that it's easy to operate it if you don't already know how to. I would need to invest time to make it easy for someone to operate it and to feel comfortable going open source and then invest the time to maintain the project. I don't have that time and I rather spend the time I have to keep what I got up to date and create weekly updates.
